Модификация структуры + 2 блока

TeG2405 opened the pull request:
Предлагаю реализацию 2-x блоков "img", "form-control", а также модификацию структуры уровней переопределения следующим образом.

Commit:
Блок form-control

Commit:
Блок form-control and img

shkarupa-alex reviewed (commented):
Нужно доделать:
-убрать уровень tags
-вернуть  1px
-вернуть дефолтные аттрибуты картинок
-переместить примеры в папку по методологии БЭМ

Commit:
Модификация структуры проекта.
Организация хранения примеров блока.
 Тестирование разметки на данном этапе техникой .enb
